email_backend/src/email_backend.js

```diff
@@ -1,41 +1,47 @@
-import express from "express";
 import body_parser from "body-parser";
 import compression from "compression";
 import cors from "cors";
-import nodemailer from "nodemailer";
+import express from "express";
 import _ from "lodash";
+import nodemailer from "nodemailer";
 
+import {
+  get_db_connection_status,
+  connect_db,
+  log_email_and_meta_to_db,
+} from "./db_utils";
 import { get_transport_config, get_email_config } from "./email_utils";
-
 import {
+  get_templates,
   validate_completed_template,
   make_email_subject_from_completed_template,
   make_email_body_from_completed_template,
 } from "./template_utils";
 
-import { log_email_and_meta_to_db } from "./db_utils";
-
 import { throttle_requests_by_client } from "./throttle_requests_by_client.js";
 
 const get_request_content = (request) =>
   (!_.isEmpty(request.body) && request.body) ||
   (!_.isEmpty(request.query) && request.query);
-
-const log_email_request = (request, log_message) => {
+const log_error_case = (request, error_message) => {
   const request_content = get_request_content(request);
-  //eslint-disable-next-line no-console
-  console.log(
-    JSON.stringify(
-      {
-        ..._.pickBy({
-          log_message,
-          request_content,
-        }),
-        sha: process.env.CURRENT_SHA || "dev, no sha env var set",
-      },
-      null,
-      2
-    )
+  console.error(
+    JSON.stringify({
+      ..._.pickBy({
+        error_message,
+        request_content,
+      }),
+      sha: process.env.CURRENT_SHA || "dev, no sha env var set",
+    })
+  );
+};
+const log_success_case = (request) => {
+  const request_content = get_request_content(request);
+  console.error(
```

Comment:
This shouldn't really be console.error, it's logging success case. Could be. .log or .info?

Reply:
Oops, typo, thanks!